src/main/java/Duke.java

```diff
@@ -1,10 +1,63 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.io.FileNotFoundException;

```

Comment:
Really nice job for coding standard! I like how clean your code is, and the Pokemon theme is really creative too :) Keep up the good job! 👍